harness/determined/pytorch/_pytorch_context.py

```diff
@@ -643,7 +656,8 @@ def backward(
 
     @staticmethod
     def _average_gradients(parameters: Any, divisor: int) -> None:
-        check.gt_eq(divisor, 1)
+        if divisor < 1:
+            raise ValueError("divisor < 1")
```

Comment:
I'd be fine if you removed this check... seems unnecessary to me.

Reply:
Removed.

Reply:
Oops, `TestPyTorchContext.test_average_gradients` actually tests for this case and for an exception being raised, so I'm going to leave this in.

Reply:
That's circular reasoning though.  There was a check that was pointless, then we wrote a test in the name of improving test coverage, now we are keeping the pointless check because we want to pass our test that was only written to improve test coverage?  That doesn't make sense.

Reply:
Gotcha. I've remove both the check and the test thereof.